example-app/config/default.json

```diff
@@ -1,4 +1,4 @@
 {
   "port": "8080",
-  "arcgisPortal": "https://www.arcgis.com"
+  "arcgisPortal": "https://qaext.arcgis.com"
```

Comment:
Did you mean to leave this change in here?

Reply:
No, nice catch!